scheduler/reconcile.go

```diff
@@ -200,30 +199,28 @@ func (a *allocReconciler) Compute() *reconcileResults {
 		return a.result
 	}
 
-	// Detect if the deployment is paused
-	if a.deployment != nil {
-		a.deploymentPaused = a.deployment.Status == structs.DeploymentStatusPaused ||
-			a.deployment.Status == structs.DeploymentStatusPending
-		a.deploymentFailed = a.deployment.Status == structs.DeploymentStatusFailed
-	}
-	if a.deployment == nil {
-		// When we create the deployment later, it will be in a pending
-		// state. But we also need to tell Compute we're paused, otherwise we
-		// make placements on the paused deployment.
-		if a.job.IsMultiregion() && !(a.job.IsPeriodic() || a.job.IsParameterized()) {
-			a.deploymentPaused = true
-		}
-	}
+	a.computeDeploymentPaused()
 
 	// Reconcile each group
+	deploymentComplete := a.computeDeploymentComplete(m)
```

Comment:
Now that we've moved `computeGroup` behind the function call, this comment isn't as clear. 

```suggestion
	// Reconcile each group to determine if the deployment is complete
	deploymentComplete := a.computeDeploymentComplete(m)
```

Alternately (maybe preferably?), just cut the comment and leave the method name to do the work of communicating what's happening.